### ISSUE TITLE
Seeder not working with nuxt-js project

### ISSUE THREAD

**Issue by SharadKumar** [thosakwe/feathers-seeder#16]
I have seeder setup with `markets` service:
```
{
      path: 'markets',
      count: 2,
      template: {
        name: '{{lorem.words(3)}}',
        slug: '{{lorem.words(1)}}'
      }
}
```
With result:
```
info: after: markets - Method: remove
info: after: markets - Method: create
info: after: markets - Method: create
```
But I see nothing show up in database (postgres). However, if I use REST api, I get it in the database.
```
info: after: markets - Method: create
```
App is otherwise working fine. Example:
- https://github.com/nuxt/nuxt.js/tree/dev/examples/with-feathers
- https://github.com/lewebsimple/feathers-nuxt

**Comment by thosakwe** [comment]
I don't think `faker` allows you to invoke functions like that. Instead, use a function:

```js
const faker = require('faker');

const seederConfig = {
  services: [
    {
      path: 'markets',
      count: 2,
      template: {
        name: () => faker.lorem.words(3),
        slug: () => faker.lorem.words(1)
      }
    }
  ]
};
```

**Comment by SharadKumar** [comment]
Thats not the problem, it made no difference with this approach of invoking. I could see faker working with previous approach on console, but data is not persisting when called by seeder and fails silently. 
So it continues to fail even after I tried above.

**Comment by thosakwe** [comment]
Not sure the problem is within the seeder. This package just calls remove (unless you tell it not to), then generates data and calls create.

The seeding function returns a Promise that you can catch; it shouldn't fail silently. 

Also, I have no experience with nuxt.js whatsoever. I hadn't heard of it until seeing this issue.

Try running the seed with "DEBUG" in your environment set to "feathers-seeder".

**Comment by SharadKumar** [comment]
Thanks for opening the issue @thosakwe 

In order to simplify for you to see the issue, I forked and prototyped the problem. I used default Nedb as well as Postgres (in use on my project)... and I can see the seeding worked with Nedb (markets model) but not Postgres (products model).

https://github.com/SharadKumar/feathers-nuxt/commit/951fe6403db558eafe931c261543ad256702ecfb

Any help is sincerely appreciated. Really keen to make seeder work, cause I am using Postman for now.
